dspace-sword/src/main/java/org/purl/sword/client/ClientFactory.java

```diff
@@ -24,8 +22,6 @@ public ClientFactory() {
         // configure the logger from the property file. The GUI client will
         // reload these properties if it is set to capture the output and
         // display it in a panel.
```

Comment:
@mwoodiupui : It looks like we should remove this entire comment section too, since we removed the code that it refers to.

Reply:
Yes, indeed.  Actually, we can delete the entire surrounding method, since it's an empty 0-argument constructor.  Done.